account_cash_flow/models/cash_flow_report.py

```diff
@@ -176,8 +176,17 @@ def calculate_moves(self):
         ])
         moves = []
         for move in moveline_ids:
+
+            # Essas variáveis são usadas apenas para o if
             debit = move.credit - move.credit_cash_basis
             credit = move.debit - move.debit_cash_basis
+
+            # Temporario: não mostra as linhas que estão com 'a receber' e
+            # 'a pagar' zerados
+            if not debit:
+                if not credit:
+                    continue
```

Comment:
Na próxima pode ser 
`
if not debit and not credit:
`

Reply:
Vdd
Thx